cobbler/actions/reposync.py

```diff
@@ -742,7 +733,7 @@ def create_local_file(self, dest_path: str, repo, output: bool = True):
         optgpgcheck = False
         if output:
             if repo.mirror_locally:
-                line = "baseurl=http://${http_server}/cobbler/repo_mirror/%s\n" % (repo.name)
+                line = "baseurl=http://${http_server}/cobbler/repo_mirror/%s\n" % (repo.name.strip())
```

Comment:
```suggestion
                line = "baseurl=http://${http_server}/cobbler/repo_mirror/%s\n" % (repo.name)
```

Let's don't strip on every use. Let's do that in the setter of the object name. I don't see a reason why we should allow leading or trailing spaces in object names. I even believe that this is not possible at the moment.

Reply:
@SchoolGuy Okay, I'll remove the unnecessary code.